torch/csrc/jit/passes/onnx/initializer_based_peephole.cpp

```diff
@@ -0,0 +1,147 @@
+#include <torch/csrc/jit/passes/onnx/initializer_based_peephole.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+#include <torch/torch.h>
+
+#include <c10/util/Optional.h>
+#include <algorithm>
+
+namespace torch {
+namespace jit {
+
+namespace onnx {
+using namespace ::c10::onnx;
+}
+
+std::vector<at::Tensor> getValues(
+    Node* node,
+    const ValueToParamPairMap& valsToParamsMap) {
+  size_t numInputs = node->inputs().size();
+  std::vector<at::Tensor> inputTensorValues;
+  inputTensorValues.reserve(numInputs);
+  for (auto val : node->inputs()) {
+    if (val->node()->kind() == prim::Param) {
+      auto itr = valsToParamsMap.find(val);
+      if (itr == valsToParamsMap.end()) {
+        continue;
+      }
+      inputTensorValues.push_back(itr->second.second.toTensor());
+    } else if (val->node()->kind() == onnx::Constant) {
+      inputTensorValues.push_back(val->node()->t(attr::value));
+    } else {
+      continue;
+    }
+  }
+  return inputTensorValues;
+}
+
+static void fuseConvBatchNorm(Block* b, ValueToParamPairMap& valsToParamsMap) {
```

Comment:
Add comments explaining the details of the fusion. 
* requirements for conv and bn such that they can be fused. 
* how they are fused.